packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -170,8 +171,10 @@ export class GasEstimator {
     if (this.type == NetworkType.London) {
       this.latestMaxFeePerGasGwei = (gasInfo as LondonGasData).maxFeePerGas;
       this.latestMaxPriorityFeePerGasGwei = (gasInfo as LondonGasData).maxPriorityFeePerGas;
-      // Extract the base fee from the most recent block. If the block is not available or errored then is set to 0.
-      this.latestBaseFee = Number((latestBlock as any)?.baseFeePerGas) || 0;
+      // Extract the base fee from the most recent block. If the block is not available or errored then is set to the
+      // latest max fee per gas so we still have some value in the right ballpark to return to the client implementer.
+      // Base fee is represented in Wei so we convert to Gwei to be consistent with other variables in this class.
+      this.latestBaseFeeGwei = Number((latestBlock as any)?.baseFeePerGas) / 1e9 || this.latestMaxFeePerGasGwei;
```

Comment:
this is the other main change in this file: if we cant get the block baseFeePerGas then it should default to the `latestMaxFeePerGasGwei`, rather than 0, as the previous implementation would return a very low expected cumulative gas price if the utility failed to query the gas price.